internal/install/execution/install_status.go

```diff
@@ -187,6 +199,16 @@ func (s *InstallStatus) hasFailed() bool {
 	return false
 }
 
+func (s *InstallStatus) isCanceled() bool {
```

Comment:
Refactor opportunity: this could be refactored to a general purpose method to test any state, and `isCanceled` and `isCompleted` could then be based on it to DRY things up a little.

Reply:
I like that idea